.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 
-script: GOPATH="" make test build
+script: GOPATH="" make vendor test build
```

Comment:
It should not run `go vendor` in travis. The point of the vendoring is to use the local files, and to not be dependent on the internet while building. I'll add a comment below at the `go build`.

Reply:
Controversial decision, but still makes sense for CI build. Just if the user got the repository himself, then there are hardly any problems with the Internet

Reply:
Yeah, but the vendor files are there now already; might as well use them.